.changeset/neat-cycles-walk.md

```diff
@@ -0,0 +1,6 @@
+---
+"myst-cli": minor
```

Comment:
I think that we can consider most changes to the CLI a patch until it has backwards incompatible user-facing behaviour. We aren't really encouraging/supporting imports from the CLI at the moment. Thoughts?

```suggestion
"myst-cli": patch
```

Reply:
As I removed some conversion logic, I made it a minor. Thus, I think the minor should stand unless I revert that change.

Reply:
Can we revert the change please? The effort to support backwards compatibility in this case is minor (and already done) and we can just do it on the fly for these authors.

I think that if we can support jupyterbooks at various versions without too much effort we should, even if requires a bit more effort on our part.